Jenkinsfile

```diff
@@ -26,6 +26,7 @@ node('ca-jenkins-agent') {
     // Protected branch property definitions
     pipeline.protectedBranches.addMap([
         [name: "master", tag: "latest", dependencies: ["@zowe/perf-timing": "latest"], aliasTags: ["zowe-v1-lts"]],
+        [name: "next", tag: "next", prerelease: "next", dependencies: ["@zowe/perf-timing": "latest"]],
```

Comment:
Should this be removed if the PR is being merged into master?

Reply:
It won't affect its behavior.
I'm fine either way 😋

Reply:
This was just a carry-over from the original POC. I will just delete the "next" line now.